docs: remove image with caption and tabs extended media pages (#1112)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/tabs-extended.mdx

<AnchorLinks>

<AnchorLink>Overview</AnchorLink>
<AnchorLink>Options</AnchorLink>
<AnchorLink>Design and functional specifications</AnchorLink>
<AnchorLink>Development documentation</AnchorLink>
<AnchorLink>Feedback</AnchorLink>

</AnchorLinks>

## Overview

Tabs extended uses the Carbon Design System's [tabs component](https://carbondesignsystem.com/components/tabs/usage/). On desktop, the component renders as tabs, allowing for large amounts of content to be organized using less vertical space on the page. On mobile, the component renders as an accordion, making it easy to read and access each section while scrolling.

> The tab labels may wrap up to two lines then truncated.

Reviewer: ```suggestion
The tab labels may wrap up to two lines when truncated.
```

On mobile, Tabs extended switches to an accordion for easier browsing.

> 

Reviewer: @RichKummer I don't think we need the `Behavior` heading here. But I would add captions to underneath each image.

Example of the tabs extended component, on desktop

Example of the tabs extended component, on mobile

</Column>
</Row>

## Options

> Tabs extended media is a variation that constructs its content using [Content item horizontal](/components/content-item-horizontal) with media to create an organized section optimized for content with media. This is useful especially when reducing page length is desired. Keep in mind some content will be hidden until user interaction, so they should not be critical information. 

Reviewer: @RichKummer Looking at this now, I think it would be better to include an H3 here. To match the Image page, this would be "Media".

> Tabs extended media is a variation that constructs its content using [Content item horizontal](/components/content-item-horizontal) with media to create an organized section optimized for content with media. This is useful especially when reducing page length is desired. Keep in mind some content will be hidden until user interaction, so they should not be critical information. 

Reviewer: ```suggestion
Tabs extended provides an option to include media. It uses [Content item horizontal](/components/content-item-horizontal) with media to create an organized section for displaying content with media. This is useful especially when reducing page length is desired. Keep in mind that when using tabs, some content will be hidden until the user interacts with the component, so they should not be used for information that is considered critical. 
```

> For more on Tabs extended media, you can view the [functional specs](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Tabs-extended-media) and [Web Components Storybook](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-tabs-extended-with-media--default).

Reviewer: ```suggestion
For more on Tabs extended using media, see the [functional specs](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Tabs-extended-media) and [Web Components Storybook](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-tabs-extended-with-media--default).
```

<Row>
<Column colMd={8} colLg={12}>

![Image of Tabs extended with media component](../../images/component/tabs-extended/components--tabs-extended-media-1.jpg)

</Column>
</Row>

> ### Behavior

Reviewer: @RichKummer I don't think you need a new section here for behavior because it's already covered above. You could just show the two images but add captions underneath each one. 

Tabs extended with the media option, on desktop

Tabs extended with the media option, on mobile
